stacks-signer/src/signer.rs

```diff
@@ -517,31 +517,29 @@ impl Signer {
                 sig: vec![],
             };
             self.handle_packets(stacks_client, res, &[packet], current_reward_cycle);
+        } else if block_info.valid.unwrap_or(false)
```

Comment:
I'm curious why this code path is preserved?  I thought the Stacks node miner itself would always be the signing round coordinator for a block?

Reply:
Good catch. This was just me fixing a clippy warning without looking into what the branch was. Will remove.

Reply:
@kantai could you confirm that me removing this is okay? I assumed it was just overlooked, but want to make sure we are not anticipating that signers operate as acoordinator if for some reason they never set the self.miner key?

Reply:
Removing this should be okay, yes.

Reply:
Just realized in this thought exercise...I am not properly handling DKG triggered in the middle of a current reward cycle still. A signer might try to trigger it as it should. But all other signers will ignore it as they treat the message as not coming from the miner coordinator.